### PR TITLE
M3-5521: Contextual help for Linode Create

### PR DESCRIPTION
**Pull request by bnussman**
## Description

This PR logs a Google Analytics event when a user does any of the following on `/linodes/create`
- Clicks the `Use our speedtest page` in the region section
- Clicks the SMTP guide link in the checkout sidebar
- Views the IPAM tooltip in the VLANs section

## How to test

- View the dev console and make sure the debugging Google Analytics logs an event when any of those 3 actions listed above are performed. 
Screen Shot 2021-11-23 at 3 01 57 PM

### CODE REVIEW

File: packages/manager/src/components/TextField.tsx [hunk 1]
```diff
@@ -89,6 +89,7 @@ interface BaseProps {
   optional?: boolean;
   required?: boolean;
   tooltipText?: string;
+  tooltipOnMouseEnter?: React.MouseEventHandler<HTMLDivElement> | undefined;
```
Comment (HanaXu) [inline packages/manager/src/components/TextField.tsx:92]: Can we rename this to something like `handleOnMouseEnter`?

**Review by HanaXu — APPROVED** [review]
LGTM! Just left one non-blocking comment on renaming a variable 📈

File: packages/manager/src/components/TextField.tsx (hunk 1 above)
Comment (bnussman) [reply packages/manager/src/components/TextField.tsx:92]: I think this would be misleading because that prop is for the TextField component.

**Review by codyfinn — APPROVED** [review]
🚀

**Pull request merged**